app/controllers/supervisor/tasks_controller.rb

```diff
@@ -1,8 +1,12 @@
 class Supervisor::TasksController < Supervisor::BaseController
+  authorize_resource class: "Task"
```

Comment:
```suggestion
  authorize_resource class: Task.name, only: [..]
```
Giới hạn lại e nha, các action nào đã check ở before_action authorize rồi thì không cần chạy authorize_resource nàynày